libp2p/dialer.nim

```diff
@@ -207,15 +213,16 @@ method connect*(
   self: Dialer,
   peerId: PeerId,
   addrs: seq[MultiAddress],
-  forceDial = false) {.async.} =
+  forceDial = false,
+  forceNewConnection = false) {.async.} =
```

Comment:
Nitpick, but maybe `reuseConnection = true` is better?

Reply:
It seems better, but on the other hand it is different from the existing pattern "forceDial".